src/Standards/BestIt/CodeSniffer/CodeWarning.php

```diff
@@ -40,9 +40,9 @@ class CodeWarning extends Exception
     /**
      * The erroneous token.
      *
-     * @var array|void
+     * @var array|null
      */
-    private $token;
+    private ?array $token;
```

Comment:
You must initialize that property (directly or by constructor).

```

private ?array $token = null;
```

Otherwise you may get an uninitialized error.